source/system/Numbers.ooc

```diff
@@ -100,7 +100,7 @@ LDouble: cover from long double {
 		result
 	}
 	in: func (range: Range) -> Bool {
-		this >= range min && this < range max
+		this as Int >= range min && this as Int < range max
```

Comment:
Since `range min` and `range max` is Int this should be same as previously.

However what does "as Int" mean, should it be truncated first? `this min(Int maximumValue) as Int >= range min && this as this max(Int minimumValue) < range max`